src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -27,11 +27,11 @@ async function boot(userOptions?: Partial<CircuitStartOptions>): Promise<void> {
   // Establish options to be used
   const options = resolveOptions(userOptions);
   const logger = new ConsoleLogger(options.logLevel);
-  window['Blazor'].defaultReconnectionHandler = new DefaultReconnectionHandler(logger);
-  window['Blazor']._internal.InputFile = InputFile;
+  Blazor.defaultReconnectionHandler = new DefaultReconnectionHandler(logger);
+  Blazor._internal.InputFile = InputFile;
```

Comment:
Thanks for sorting out some of this syntax weirdness! We've had an issue on the backlog for a while to come and tidy up around here in various ways, including this one.